config/config.wikimedia.yaml

```diff
@@ -125,10 +129,18 @@ services:
             # HastyProducer doesn't block HTTP clients, so we can
             # afford to wait for a largish batch size.
             queue.buffering.max.ms: 1000
+            batch.num.messages: 500
+            # Custom kafka config, will call producer.setPollInterval
+            # with this value if set.
+            producer.poll.interval.ms: 200
         guaranteed:
           conf:
             # GuaranteedProducer doese block HTTP clients, so we
             # should use a smaller batch size. Waiting 50 ms
             # will allow for some kafka produce reqeust batching
             # while minimizing latency on the HTTP client.
-            queue.buffering.max.ms: 50
+            queue.buffering.max.ms: 0
```

Comment:
This ain't match with the comment anymore. And why 0?

Reply:
0 is actually the default the librdkafka uses.  In my perf tests, this seemed to get the best throughput.  Still remains to be seen what is the best combo in prod.